drivers/clock_control/clock_control_silabs_siwx91x.c

```diff
@@ -16,10 +17,19 @@
 #include "rsi_pll.h"
 #include "clock_update.h"
 #include "sl_si91x_clock_manager.h"
+#ifdef CONFIG_WISECONNECT_NETWORK_STACK
+#include "sli_siwx917_soc.h"
+#endif
+#ifdef CONFIG_PM
+#include "sl_si91x_power_manager.h"
+#endif
 
-#define DT_DRV_COMPAT silabs_siwx91x_clock
+#define DT_DRV_COMPAT          silabs_siwx91x_clock
 #define DT_DRV_COMPAT          silabs_siwx91x_clock
 #define LF_FSM_CLOCK_FREQUENCY 32768
+#ifdef CONFIG_PM
+#define M4SS_RAM_SIZE_KB (DT_REG_SIZE(DT_NODELABEL(sram0)) / 1024)
+#endif
```

Comment:
There is no reason to place #ifdef around this definition.

BTW, does it make sense to introduce this intermediate symbol? A local variable with the same name will do the job. so you won't have to pollute the global namespace.